src/Categories/NaturalTransformation/NaturalIsomorphism.agda

```diff
@@ -71,6 +71,31 @@ record NaturalIsomorphism {C : Category o ℓ e}
       }
     }
 
+-- Like IsIso for morphisms, a natural transformation may be an isomorphism
+-- if there exists some inverse to it.
+-- To be consistent, there's also an IsNI for natural isomorphisms
+record IsNI {C : Category o ℓ e}
+            {D : Category o′ ℓ′ e′}
+            {F G : Functor C D}
+            (F⇒G : NaturalTransformation F G) : Set (o ⊔ ℓ ⊔ ℓ′ ⊔ e′) where
+  field
+    inv : NaturalTransformation G F
+
+  module F⇒G = NaturalTransformation F⇒G
+  module inv = NaturalTransformation inv
```

Comment:
For consistency with NaturalIsomorphism`, I think `inv` should be called `F⇐G` (everywhere in `IsNI`).

Reply:
Thank you for your review!

It's true that `F⇐G` is more consistent with NaturalIsomorphism; a concern here is that the name `inv` is taken from `IsIso`, and should `IsNI` try to be consistent with `IsIso`... ?

Reply:
It's a tough call. I think the instance of `NaturalIsomorphism` below 'wins' as far as consistency is concerned.